src/components/form/checkbox/checkbox.test.tsx

```diff
@@ -61,6 +61,13 @@ describe('EuiCheckbox', () => {
       expect(component).toMatchSnapshot();
     });
 
+    test('labelProps is rendered', () => {
+      const component = render(
+        <EuiCheckbox {...checkboxRequiredProps} labelProps={requiredProps} />
```

Comment:
Looks like in order for the snapshot to render the `labelProps` you need to also pass a label
```suggestion
        <EuiCheckbox {...checkboxRequiredProps} label="Label" labelProps={requiredProps} />
```